aspnet/client-side/yeoman.rst

```diff
@@ -1,95 +1,92 @@
 
-.. note:: The ``–g`` flag installs the generator globally, so that it can be used from any path on your system.
+- 	- The ``–g`` flag installs the generator globally, so that it can be used from any path.
+
+Create an ASP.NET app
+-------------------------
  
-3. Create the new directory in which the project will be generated:
+- Create a directory for the project
 
 .. code-block:: console
 
  mkdir C:\MyYo
-
-4. On the command line, make the new directory the current directory:
-
-.. code-block:: console
- 
  cd C:\MyYo
 
-5. Run the ``yo`` command, providing the ``aspnet`` alias for the ASP.NET generator:
+- Run the ASP.NET generator for ``yo``
 
 .. code-block:: console 
 
  yo aspnet
-
-6. The generator displays a menu. Arrow down to the **Web Application** project type and press ``Enter`` to select it:
+ 
+- The generator displays a menu. Arrow down to the **Web Application** project and tap **Enter**:
 
 .. image:: yeoman/_static/yeoman-yo-aspnet.png
 
-7. Provide an application name of "MyWebApp" and press ``Enter``:
+- Use "MyWebApp" for the app name and then tap **Enter** :
 
-.. image:: yeoman/_static/yeoman-yo-aspnet-appname.png
+	.. image:: yeoman/_static/yeoman-yo-aspnet-appname.png
 
-Yeoman will scaffold the project and its supporting files. Suggested next steps are also provided in the form of commands. 
+	Yeoman will scaffold the project and its supporting files. Suggested next steps are also provided in the form of commands. 
 
-.. image:: yeoman/_static/yeoman-yo-aspnet-created.png
+	.. image:: yeoman/_static/yeoman-yo-aspnet-created.png
 
-Client-Side Build Support
--------------------------
+The `ASP.NET generator <https://www.npmjs.com/package/generator-aspnet>`__ creates ASP.NET 5 DNX projects that can be loaded into Visual Studio 2015 or run from the command line. 
+
+If you were redirected to this tutorial from :doc:`/tutorials/your-first-mac-aspnet`, you can return now.
+
+The `ASP.NET generator <https://www.npmjs.com/package/generator-aspnet>`__ creates the following project types:
+
+- **Empty Application**: An empty Web application with minimal dependencies. 
+- **Console Application**: A DNX-based console application.
+- **Web Application**: A complete MVC web application with a simple home page and examples for managing accounts and login.
+- **Web Application Basic**: A simpler MVC web application lacking membership and authorization, among other advanced features which are found only in the **Web Application** generator. 
+- **Web API Application**: A Web API built with MVC. 
+- **Nancy ASP.NET Application**: A lightweight HTTP service with one module built using `Nancy <http://nancyfx.org/>`_.
+- **Class Library**: A DNX-based class library.
+- **Unit Test project**: A simple `XUnit <http://xunit.github.io/>`_ test project. 
+
+Setting Grunt as the task runner
+---------------------------------
+
+:doc:`Grunt </client-side/using-grunt>`
```

Comment:
Remove these references and instead inline them where Gulp and Grunt are first mentioned.